packages/utils/src/test/twitter/twitter-utils.test.ts

```diff
@@ -0,0 +1,95 @@
+import { describe, expect, test } from "@jest/globals"
```

Comment:
Did you test it? Does it work?

Reply:
Yes, it works.

![Screen Shot 2023-01-26 at 15 12 39](https://user-images.githubusercontent.com/14356826/214787088-069199b6-a2aa-4aee-a63c-21812d9ba61b.png)